sdk/python/examples/client-simple/README.md

```diff
@@ -0,0 +1,27 @@
+# Simple example using the Python SDK client
+
+## Requirements
+
+- Python 3.10+
+- [Docker](https://docs.docker.com/engine/install/)
+
+## How to use?
+
+```sh
+python3 -m venv .venv
+source .venv/bin/activate
+pip install --pre dagger-io
+python say.py "Simple is better than complex"
```

Comment:
That took a long time to return the first time (`21s`). Second run is 10x quicker (`2s`). Some feedback during those initial `21s` would have helped. Not an issue, just mentioning it.

<img width="691" alt="image" src="https://user-images.githubusercontent.com/3342/201131583-7985327a-2993-4ce9-ba98-750e6fd8c3fd.png">

Reply:
![say-status](https://user-images.githubusercontent.com/174525/201230373-861cc7bc-13fd-47ef-8dd4-811e9e48ea2a.gif)